.github/workflows/_linux-build.yml

```diff
@@ -187,7 +187,7 @@ jobs:
       - name: Archive artifacts into zip
         if: inputs.build-generates-artifacts && steps.build.outcome != 'skipped'
         run: |
-          zip -1 -r artifacts.zip dist/ build/custom_test_artifacts build/lib build/bin .additional_ci_files
+          sudo zip -1 -r artifacts.zip dist/ build/custom_test_artifacts build/lib build/bin .additional_ci_files
```

Comment:
We need to check it passes on other linux runners like aarch64.

Reply:
what labels do I need to include in order to confirm this?

Reply:
@jeanschmidt ciflow/binaries_wheel to trigger aarch64 wheel builds

Reply:
I don't know what you guys are trying to do here, but this is very/very/very wrong. One should not assume that they have `sudo` privileges while running the actions

Reply:
removed, seems that after all the changes made in the arc runner itself, the sudo here was not needed anymore